cmd/jaeger/integration/receivers/storagereceiver/reciver.go

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
what is this package for?

Reply:
I removed it. I had thought it was needed for Badger, but it works fine without it.